src/rnn.cpp

```diff
@@ -845,8 +845,8 @@ void RNNDescriptor::GetLayerBiasOffset(const int layer,
         return;
     }
 
-    auto bdim = int(hsize);
-    biasDesc  = miopen::TensorDescriptor(dataType, &bdim, 1);
+    auto bdim = hsize;
+    biasDesc  = miopen::TensorDescriptor(dataType, {bdim});
```

Comment:
@atamazov Since `hsize` is of type `size_t`, `bdim` is also of this type, `{bdim}` is of type `std::initializer_list<size_t>`. In this case, the 3rd constructor is called.